### PR TITLE
exercise is finished

### PR DESCRIPTION
**Pull request by pattycsouza**

### CODE REVIEW

File: day_1/day_1_pattyexercise.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+# Print two numbers being multiplied
+# Print a string being multiplied
+# Your code goes below this line
+
+name = 'Patricia'
+lastname = 'Souza'
+age = 31
+weight = 81.5
+
+print (name +' '+ lastname)
```
Comment (ajditto) [inline day_1/day_1_pattyexercise.py:28]: Not required, but you should usually leave a space between operators (ie: (name + ' ' + lastname))

File: day_1/day_1_pattyexercise.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+print (age*weight)
```
Comment (ajditto) [inline day_1/day_1_pattyexercise.py:29]: Same thing ere (age * weight)

File: day_1/day_1_pattyexercise.py [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+print (name*eight)
```
Comment (ajditto) [inline day_1/day_1_pattyexercise.py:30]: It's not required, but generally you should leave a space between operations but not functions, so these 3 lines would look like this:
print(name + ' ' + lastname)
print(age * weight)
print(name * eight)

**Review by ajditto — APPROVED** [review]

**Comment by ajditto** [comment]
Looks great!

File: day_1/day_1_pattyexercise.py (hunk 3 above)
Comment (pattycsouza) [reply day_1/day_1_pattyexercise.py:30]: ty @ajditto , it really makes the code "prettier" 😃

**Pull request merged**